### PR TITLE
Reduce the size of vicadmin footprint

### PR DESCRIPTION
**Pull request by kjplatz**
Change the embedded fonts from the full fontawesome fonts to a custom fontello font suite which includes only the 3 glyphs we need.  This reduces the vicadmin footprint by over 120k.

### CODE REVIEW

**Comment by hmahmood** [comment]
lgtm

**Comment by matthewavery** [comment]
lgtm

File: isos/vicadmin/css/fontello.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  src: url('/fonts/fontello.eot?26878433');
+  src: url('/fonts/fontello.eot?26878433#iefix') format('embedded-opentype'),
+       url('/fonts/fontello.woff2?26878433') format('woff2'),
+       url('/fonts/fontello.woff?26878433') format('woff'),
+       url('/fonts/fontello.ttf?26878433') format('truetype'),
+       url('/fonts/fontello.svg?26878433#fontello') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+/* Chrome hack: SVG is rendered more smooth in Windozze. 100% magic, uncomment if you need it. */
```
Comment (gigawhitlocks) [inline isos/vicadmin/css/fontello.css]: s/Windozze/Windows

**Comment by gigawhitlocks** [comment]
one nitpick, otherwise LGTM

**Comment by kjplatz** [comment]
bing

**Pull request merged**